src/fa/check.py

```diff
@@ -24,7 +24,13 @@ def map_(mapname, force=False, silent=False):
     if fa.maps.isMapAvailable(mapname):
         logger.info("Map is available.")
         return True
-
+ 
+    if "neroxis_map_generator" in mapname:
+        import client # FIXME
```

Comment:
fix

Reply:
If importing at the top does not cause issues, do it. If it does, leave it as it is - no point trying to fix client singletons when it's on its way out.

Reply:
We have found the way